modules/installation-and-upgrade/pages/server-minor-upgrade-uyuni.adoc

```diff
@@ -44,6 +44,11 @@ zypper ref
 ----
 zypper up 
 ----
+[NOTE]
+====
+Uyuni is different from SUSE Manager in this step.  SUSE Manager uses 'zypper patch', but Uyuni requires 'zypper up'
+====
+
 +
```

Comment:
This is WIP.  Proper ifdef will follow.  ATM, github does not let me a proper patch...

```suggestion
----
zypper patch
----
----
zypper up
----
[NOTE]
====
Uyuni is different from SUSE Manager in this step.
SUSE Manager uses 'zypper patch', but Uyuni requires 'zypper up'.
====

+
```

Reply:
I don't think ifdef is needed here. Please check the filename. Seems to be 100% specific to Uyuni.

Reply:
Ok, the less ifdefs the better!  I adjusted my suggestion to a minimum.